turbinia/tcelery.py

```diff
@@ -55,12 +55,8 @@ def setup(self):
     self.app.conf.update(
         task_default_queue=config.INSTANCE_ID,
         accept_content=['json'],
-        # TODO(ericzinnikas): Without task_acks_late Celery workers will start
-        # on one task and prefetch another (i.e. can result in 1 worker getting
-        # 2 plaso jobs while another worker is free). But enabling this causes
-        # problems with certain Celery brokers (duplicated work).
```

Comment:
Do we know if this "duplicated work" would affect us or what conditions that would occur under?

Reply:
To my understanding the duplicated work only occurs when a Celery workers crashses/Task had failed then data would be reprocessed again, which I believe in our case shouldn't matter. 

I believe I remember putting a debugger in and tried looking for additional Tasks queued by the celery worker and wasn't able to identify any. Can relook into this further (if it's possible to look at Celery Task queue  from debugger midst it running) to confirm.

Reply:
@aarontp ty for review, addressed your comments and had a follow up or two. Please lmk of any questions